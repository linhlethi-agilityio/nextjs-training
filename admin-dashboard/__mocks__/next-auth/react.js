export const useSession = () => ({
  data: {
    user: {
      token: 'access-token',
      email: 'user@gmail.com',
      id: '1',
      name: 'User',
      role: 'admin',
    },
    expires: Date.now().toString(),
  },
  status: 'authenticated',
  update: async () => null,
});
