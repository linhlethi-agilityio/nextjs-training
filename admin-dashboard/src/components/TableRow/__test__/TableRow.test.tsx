import { render } from '@testing-library/react';
import { Table, Tbody, Td } from '@chakra-ui/react';

// Components
import { TableRow } from '..';

describe('TableRow', () => {
  it('should match snapshot for TableRow', () => {
    const handleClickRow = jest.fn();

    const { container } = render(
      <Table>
        <Tbody>
          <TableRow handleClickRow={handleClickRow}>
            <Td>inches</Td>
          </TableRow>
        </Tbody>
      </Table>,
    );

    expect(container).toMatchSnapshot();
  });
});
