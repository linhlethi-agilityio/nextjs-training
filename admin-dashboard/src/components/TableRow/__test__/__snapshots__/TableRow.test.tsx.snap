// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow should match snapshot for TableRow 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <tbody
      class="css-0"
    >
      <tr
        class="css-4g6ai3"
      >
        <td
          class="css-0"
        >
          inches
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
